### ISSUE TITLE
Export SliderComponent in public API so we can benefit from TypeScript when using @ViewChild

### ISSUE THREAD

**Issue by eulersson** [angular-slider/ngx-slider#294]
At the moment I cannot programatically set the slider to disabled, and updating the `Options` the slider is constructed with won't make it react, so I have to do something like this:

```typescript
@Component({
  selector: 'mycomp',
  template: `<button (click)="disableSlider()">Disable Slider</button><ngx-slider #mySlider ...></ngx-slider>`
  ...
})
export class MyCompComponent {
    @ViewChild('mySlider') sliderComponent!: any; // <-- I would like to use SliderComponent from this tool package.

    disableSlider() {
      this.sliderComponent?.setDisabledState(true);
    }
    
}
```

I have to use `any` because the `SliderComponent` class isn't exported in the [public api file](https://github.com/angular-slider/ngx-slider/blob/master/src/ngx-slider/lib/public_api.json).

Is there any reason why the class isn't exported?

I'm happy to contribute and do it.

**Comment by piotrdz** [comment]
Merged to master, will be released in next NPM package version.

**Comment by piotrdz** [comment]
Released in v17.0.0.
